services/api/db/models.py

```diff
@@ -0,0 +1,66 @@
+from django.db import models
+from django.contrib.auth.models import BaseUserManager,AbstractBaseUser, PermissionsMixin
+
+#  Custom User Manager
+class UserManager(BaseUserManager):
+    def create_user(self, email, name, password=None, **kwargs):
```

Comment:
is name the optional field if not then may be we can store some default value or throw error the way it has been done for email

Reply:
email is default thing , because we will be generating user name from email in github/google like from pandeyshikhar20@gmail.com somthing like shikhar98 or etc..thats why why i kept email as primary thing and we have ID which i have not implemented it yet (uuid based).

Reply:
name is not mandatory thing or may be first name and last name will also come so it will change depends upon front end integration.

Reply:
it will change later for sure basis on front end demand, there is no need to do redo work , I'm closing it right now.

Reply:
![image](https://github.com/kalvilabs/kalvios/assets/142379213/d11defe4-e5fa-4092-8d8b-868fb431179b)
The name field seems to be a mandatory field, so any one of the following may be opted:-
a. set null = True (seems not to be desirable)
b. add the validation for name as well like email
c. set some default value for name

Reply:
![image](https://github.com/kalvilabs/kalvios/assets/142379213/8b200112-eaeb-4b43-a13d-abb9eec1f0d2)

Reply:
we will modify this is integration, i have added to do list